test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6518,6 +6518,92 @@ def forward(self, y):
                       input_names=["input_1"],
                       dynamic_axes={"input_1": [0, 1]})
 
+    # onnx::Identity of sequence supported for ONNX opset >= 14
+    @skipIfUnsupportedMinOpsetVersion(14)
+    @skipIfONNXShapeInference(False)
+    def test_uninitialized_tensorList(self):
+        class UninitializedTensorListModel(torch.nn.Module):
+            def forward(self, x):
+                if x[0].shape[0] < 5:
+                    if x.size(0) == 1:
+                        x = x + 4
+                    else:
+                        return [x]
+                return [x]
+
+        x = torch.ones((3, 4), dtype=torch.int)
+        self.run_test(torch.jit.script(UninitializedTensorListModel()), x)
+
+    # onnx::Identity of sequence supported for ONNX opset >= 14
+    @skipIfUnsupportedMinOpsetVersion(14)
+    @skipIfONNXShapeInference(False)
+    def test_uninitialized_tensorList_dynamic(self):
+        class UninitializedTensorListModel(torch.nn.Module):
+            def forward(self, x):
+                if x[0].shape[0] < 5:
+                    if x.size(0) == 1:
+                        x += x
+                    else:
+                        return list(x)
+                return list(x)
+
+        x = torch.ones((3, 4), dtype=torch.double)
+        self.run_test(torch.jit.script(UninitializedTensorListModel()), x, input_names=["input_1"],
+                      dynamic_axes={"input_1": [0, 1]})
+
+    # onnx::Identity of sequence supported for ONNX opset >= 14
+    @skipIfUnsupportedMinOpsetVersion(14)
+    @skipIfONNXShapeInference(False)
+    def test_uninitialized_intList(self):
```

Comment:
hi @hwangdeyu , this test starts to fail after syncing with latest master + onnx_ms_1 branch. Please take a look when you get a chance. I'll submit the PR to pytorch master with this test disabled.

Reply:
filed a work item: https://msdata.visualstudio.com/Vienna/_workitems/edit/1441852

Reply:
update: we discovered it is due to upstream torch IR graph changes that led to the regression. The initial int list creation is noe emitted as `prim::ListConstruct` instead of `prim::Loop`.
Test case is updated to preserve the previous expected IR graph. The item above can be used to track future support for the new IR graph.